src/herbpy/tsr/fuze.py

```diff
@@ -0,0 +1,55 @@
+import numpy
+from prpy.tsr.tsrlibrary import TSRFactory
+from prpy.tsr.tsr import *
+
+@TSRFactory('herb', 'fuze_bottle', 'point')
+def point_at_obj(robot, bottle, manip=None):
+    '''
+    @param robot The robot performing the point
+    @param bottle The bottle to point
+    @param manip The manipulator to perform the grasp. if None 
+        the active manipulator on the robot is used
+    '''
+
+    if manip is None:
+        manip_idx = robot.GetActiveManipulatorIndex()
+    else:
+        with manip.GetRobot():
+            manip.SetActive()
+            manip_idx = manip.GetRobot().GetActiveManipulatorIndex()
+		
+    # compute T_ow
+    T0_w_0 = bottle.GetTransform()
+    T0_w_1 = numpy.identity(4)
+
+    # compute T_we. Tuned once and used forever onward 
+    # Insures pointer points at object with right arm
```

Comment:
This was a tad careless of me. 

This actually only works on the right arm. Pointing from anything other then an arm doesnt make sense and this is actually tuned to right arm. What's the stylistically koval-approved way for setting it to be the right arm?